src/api/tenderly/tenderlyApi.ts

```diff
@@ -70,11 +70,14 @@ export function traceToTransfersTrades(trace: Trace): TxTradesAndTransfers {
   try {
     trace.logs.forEach((log) => {
       if (log.name === TypeOfTrace.TRANSFER) {
+        const from = log.inputs[IndexTransferInput.from].value
+        const to = log.inputs[IndexTransferInput.to].value
         transfers.push({
           token: log.raw.address,
-          from: log.inputs[IndexTransferInput.from].value,
-          to: log.inputs[IndexTransferInput.to].value,
+          from,
+          to,
           value: log.inputs[IndexTransferInput.value].value,
+          isInternal: from === to,
```

Comment:
Assuming only internal trades will ever have same from & to

Reply:
Technically, im not even sure is the case

It would be weird that there's an order under this case, it means we spend gas just to do nothing. And we make the user pay some gas. So doesn't look like a legit case, and I don't think the protocol should support wash-trading

Reply:
so im fine with this assumption

Reply:
I moved the node somewhere else, to be clearer there's a trade involved.

Still, if not `isInternal`, what's your suggestion?